lib/services/registered_addons.pm

```diff
@@ -33,6 +33,7 @@ sub suseconnect_ls {
 sub check_registered_system {
     my ($system) = @_;
     my $pro = uc get_var('SLE_PRODUCT');
+    $pro = 'SLE_' . $pro if ($pro eq 'HPC');
```

Comment:
better using $pro =~ /hpc/i

Reply:
I only need match HPC. As I need add SLE_ before HPC, So It can be same with SUSEConnect output. Any other even include hpc cannot match this condition.